docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+
+> Below, days mean business days, so if the time period includes the
+> weekend, add two days, and for major holidays add a day.
+> Furthermore, most of us are academics, and we occasionally go to
+> conferences which may lead to delays in the review process.  Also do
+> not expect much to happen between December 20th and January 3rd.
+
+Most pull requests submitted to SpECTRE will be reviewed in the
+following manner:
+- When creating the pull request, the author should add the
+  appropriate labels.  If the pull request is not a `new design` or
+  `in progress`, the author may either assign two reviewers (if GitHub
+  suggested any) or add the `review help wanted` label.
+- Within two days, assigned reviewers should either confirm that they
+  are able to review, or remove themselves while adding the `review
+  help wanted` label.
+- Anyone is welcome to self-assign themselves as a reviewer; when
+  there are at least two reviewers, the `review help wanted` label
+  should be removed.
+- If there are not two reviewers after two days, a SpECTRE core
+  developer will assign reviewers, removing the `review help wanted`
+  label.  If for some reason, no reviewers have been assigned after
+  three days, the author of the pull request should feel free to ping
+  `@sxs-collaboration/spectre-core-devs`
+- Anyone can request changes within the first two days.  After two
+  days only the assigned reviewers can request changes, unless someone
+  believes the code is wrong.  Non-reviewers are allowed to make
+  comments, which the pull request author is encouraged to address,
+  but alternatively can create an issue with the suggested changes,
+  which will be addressed in a subsequent pull request by the author.
+- Assigned reviewers should submit their review in as timely a manner
+  as possible.
+- If any requested change is unclear to the author, they should ping
+  the reviewer and ask for clarification.  If needed, the author and
+  reviewers should talk to one another (in person, via Google hangout,
+  ...) to resolve any issues.
+- Reviewers are encouraged to ping others to get opinions on code they
+  are unsure about.
+- It is permissible to have a group code review led by one of the
+  reviewers. The reviewer should comment on who was present at the
+  group review.
+- If necessary, pull requests can also be discussed at one of the
+  weekly SpECTRE meetings.
+- If changes are requested, the author should fix all of them in one
+  or more fixup commits, and then ping the reviewers that the pull
+  request is updated.  In addition the `updated` label can be added.
+- Once a pull request is updated, the reviewers should either request
+  further changes (removing the `updated` label) or tell the pull
+  request author to squash their commits.
+- Once the commits are squashed, the author should ping the reviewers.
+- Once all reviewers have approved the pull request, someone should
+  ping the `@sxs-collaboration/spectre-core-devs`.
```

Comment:
You could add here that the ping is not necessary if one of the reviewers is a member of that group already (unless you want the ping anyway, I would be fine with that). However, I don't think we should require a _second_ member of the core-devs group to add a review.